public/sass/pages/_explore.scss

```diff
@@ -140,6 +140,10 @@
 .cheat-sheet-item__example {
   margin: $space-xs 0;
   cursor: pointer;
+  // element is interactive, clear button styles
```

Comment:
It looks like there might be a couple of places this classname is used on divs still besides in Cloudwatch. Were you able to double check if adding these additional style properties is a problem for them? I assume it's fine but would be good to double check! We could also change the divs to buttons wherever they are used and make this change in one fell swoop if we want!

Alternatively, I see we've been using a tool called [Emotion](https://emotion.sh/docs/introduction) to generate classnames and keep them focused where the component is. Here's an example https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/cloudwatch/components/XrayLinkConfig.tsx#L32
In this way we could leave this class untouched for anyone still using it, and create a unique classname that is only used in cloudwatch.

Reply:
oh also found this styleguide which might be a better guide for using emotion with grafana https://github.com/grafana/grafana/blob/main/contribute/style-guides/styling.md

Reply:
thought of yet another possibility of things to consider (sorry for being nitpicky! mostly just sharing options for onboarding purposes haha)

We have a [Button](https://developers.grafana.com/ui/latest/index.html?path=/story/buttons-button--basic&args=variant:secondary;fill:text) component without a capital "B" that lives in @grafana/ui you could try to see if that helps potentially!

Reply:
- I've decided to keep the styles in the existing class, since this one always has to be a <button> element for a11y purposes, so we dont need a special one without the styles. Regarding the <Button> element from UI components, I decided to not use since it would need additional styling just to make it less <Button/>-y and to fit it in the page nicely. 
- Made changes for all components now, thanks for the suggestion!